### ISSUE TITLE
Datepicker filter does not send payload on mobile resolution

### ISSUE THREAD

**Issue by matt-tolliday-iss** [outfielddigital/konstrukt#46]
**Describe the bug**
When viewing the grid on a mobile resolution, or mobile simulation (Chrome DEV tools tested with) then the date pickers do not get sent with the payload to the FindEntities method, so the grid data set does not get filtered.

`.AddFilterableProperty(p => p.LastLogin, filterconfig => filterconfig
                        .SetLabel("Last Logged In")
                        .SetDescription("When the subscriber last logged into the newsletter site")
                    )`

**Steps To Reproduce**
1. Activate Chrome Dev Tools - set mobile device to iPhone 12 Pro
2. Click on filter funnel and set date range filter
3. Apply
4. Data not filtered

**Expected behavior**
Data filtered as expected by range filter

**Screenshots**
Payload after filter applied (Chrome Dev: :iPhone 12 Pro)
<img width="605" alt="image" src="https://user-images.githubusercontent.com/80101704/212130496-518ebf3f-0b39-419d-8a09-f77131950282.png">

Payload at standard browser res:
<img width="581" alt="image" src="https://user-images.githubusercontent.com/80101704/212130728-91bf895f-4983-43e8-91d1-e82c48e25d9f.png">

Payload before filter applied:
<img width="605" alt="image" src="https://user-images.githubusercontent.com/80101704/212130578-bf008aa6-fbab-41b4-b55a-05f3c464ba7d.png">

**Environment (please complete the following information):**
 - Server OS: [e.g. iOS]
 - Browser Chrome
 - Umbraco Version 10.3.2
 - Product Version 1.6.3-beta0010

**Additional context**

**Comment by mattbrailsford** [comment]
So that's a weird on, but I think it should be working now. A new build should be on the unstable feed shortly if you want to test.

**Comment by matt-tolliday-iss** [comment]
Yup, had to reset the Runtime Minification Version - but assets loaded now and all works correctly, thanks

**Comment by mattbrailsford** [comment]
Fixed in 1.6.3
